Assets/MRTK/SDK/Features/Utilities/Migration/ButtonConfigHelperMigrationHandler.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.UI;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities
+{
+    /// <summary>
+    /// Migration handler for migrating buttons with custom icons to the button config helper.
+    /// </summary>
+    public class ButtonConfigHelperMigrationHandler : IMigrationHandler
+    {
+        public bool CanMigrate(GameObject gameObject)
+        {
+#if UNITY_EDITOR
+            ButtonConfigHelper bch = gameObject.GetComponent<ButtonConfigHelper>();
+            return bch != null && bch.EditorCheckForCustomIcon();
+#else
+            return false;
```

Comment:
@vaoliva @Railboy I don't fully understand why there's a non-Unity Editor case - I assumed that these migration handlers were editor-only in the first place (i.e. should we actually just have this asmdef marked as editor only instead of doing these types of ifdefs?

Reply:
The EditorCheckForCustomIcon method is only defined in editor, and the CanMigrate method has to return a value, so in the non-editor version I just return false. Seemed like the least contrived way to handle it.

Using #if UNITY_EDITOR seems pretty common in these migration handlers but I'm not sure what thinking led that to be the case.